packages/oui-autocomplete/src/autocomplete.controller.js

```diff
@@ -232,6 +234,12 @@ export default class {
                 .attr("autocomplete", "off")
                 .attr("list", this.id)
                 .one("focus", () => this.createDatalist()) // One time bind to create the popper helper
+                .on("focus", () => {
```

Comment:
Can't you use `ngFocus` instead ? I don't know what it better 😄

Reply:
I keep the same logic, see the line before and after my update ;)